pyblazing/pyblazing/apiv2/context.py

```diff
@@ -1050,7 +1278,59 @@ def unify_partitions(self, input):
             return input
 
 
-    def sql(self, sql, table_list=[], algebra=None, use_execution_graph = False):
+    def sql(self, sql, table_list=[], algebra=None, use_execution_graph=False):
+        """
+        Query a BlazingSQL table. 
+        
+        Returns results as cudf.DataFrame on single-GPU or dask_cudf.DataFrame when distributed (multi-GPU).
+        
+        Parameters
+        ----------
+        
+        sql : string of SQL query.
+        algebra (optional) : string of SQL algebra plan.
```

Comment:
mention that if you use the algebra parameter, it will not use the sql string

Reply:
https://github.com/gumdropsteve/blazingsql/commit/948e0f8f5a0dc684715830af12c2356286327f7f#diff-b733e955c1d24a9bd95af9c95b18d70fR1311